lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -38,16 +38,17 @@ def paintEvent(self, event):
 
         painter = QtGui.QPainter(self)
 
-        rect = event.rect()
-        left = rect.left()
-        top = rect.top()
-        width = rect.width()
-        height = rect.height()
         # See documentation of QRect: bottom() and right() are off by 1, so use
         # left() + width() and top() + height().
-        bbox = Bbox(
-            [[left, self.renderer.height - (top + height * self._dpi_ratio)],
-             [left + width * self._dpi_ratio, self.renderer.height - top]])
+        rect = event.rect()
+        width = rect.width() * self._dpi_ratio
+        height = rect.height() * self._dpi_ratio
+        left, top = self.mouseEventCoords(rect.topLeft())
```

Comment:
Can you leave a comment in the code about why we are calling this?

Reply:
Yeah for sure. Good call.